Formula/dvm.rb

```diff
@@ -0,0 +1,35 @@
+class Dvm < Formula
+  desc "Docker Version Manager"
+  homepage "https://github.com/getcarina/dvm"
+  url "https://github.com/getcarina/dvm.git",
+        :tag => "0.6.2",
+        :revision => "6467884b2e25d080c25aa647bbcbbf8c12dc45bd"
+  head "https://github.com/getcarina/dvm.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    dvmpath = buildpath/"src/github.com/getcarina/dvm"
+    dvmpath.install Dir["{*,.git}"]
+
+    cd dvmpath do
+      system "make", "VERSION=#{version}", "UPGRADE_DISABLED=true"
+
+      prefix.install "dvm.sh"
+      (prefix/"dvm-helper").install "dvm-helper/dvm-helper"
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    dvm is a shell function, and must be sourced before it can be used.
+    Add the following command to your bash profile:
+      . "$(brew --prefix dvm)/dvm.sh"
+    EOS
+  end
+
+  test do
+    system "bash -c 'source $(brew --prefix dvm)/dvm.sh && dvm --version'"
```

Comment:
Might be nice to check the version output is correct, just as a teeny bit more rigorous a test:
```ruby
test do
  output = shell_output("bash -c 'source $(brew --prefix dvm)/dvm.sh && dvm --version'")
  assert_match version.to_s, output
end
```

(I didn't test this, but it should pass)